website/settings/defaults.py

```diff
@@ -1782,3 +1787,22 @@ def parent_dir(path):
 
 # refresh campaign every 5 minutes
 CAMPAIGN_REFRESH_THRESHOLD = 5 * 60  # 5 minutes in seconds
+
+# sitemap default settings
+
+SITEMAP_URL_MAX = 25000
+SITEMAP_INDEX_MAX = 50000
+SITEMAP_STATIC_URLS = [
```

Comment:
Not sure it matters, but the part that iterates over these was removed.  It is in the django version.